src/command_modules/azure-cli-rdbms/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.0.13
```

Comment:
Does this have breaking changes relative to the old model?  if so, you should bump the minor version.

Reply:
Yes, this one will run against new API.